wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Mentions/UserSearchResultsViewController.swift

```diff
@@ -268,7 +268,12 @@ extension UserSearchResultsViewController: UICollectionViewDataSource {
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let user = searchResults[indexPath.item]
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: UserCell.reuseIdentifier, for: indexPath) as! UserCell
-        cell.configure(with: user, selfUser: ZMUser.selfUser())
+        let selfUser = ZMUser.selfUser()
+        if let selfUser {
```

Comment:
Is there a difference to this?
```suggestion
        if let selfUser = ZMUser.selfUser() {
```

Reply:
No, changed it everywhere: c58dc22f24bca16acdebc09585e0df9e036f3b3f